components/ILIAS/Mail/classes/Mime/Transport/class.ilMailMimeTransportBase.php

```diff
@@ -116,7 +115,7 @@ final public function send(ilMimeMail $mail): bool
             }
         }
 
-        if ($mail->getFinalBodyAlt() !== '') {
+        if ($mail->getFinalBodyAlt()) {
```

Comment:
We don't need these changes for the feature, right? We should keep the PR minimal.